### ISSUE TITLE
Support for Android events

### ISSUE THREAD

**Issue by andrewhr** [ReactiveX/RxJava#335]
Hello,

**TL;DR: I want to contribute on Android support**

In the past few weeks I'm become very interested in FRP in general, and how it applies to better wire code in GUI applications - my main interest is to build more elegant Android code.

So, I found that here on RxJava there is an initial code to support run RxJava on Android, which is really awesome! So, I've started to work on some code to try to grasp how to work with Rx.

After that, I want to have some helper methods to transform Android event listeners into RxJava's `Subject`s, so we can really start to make it useful to cleanup our `Activity`/`Fragment` code.

There is something like that waiting to be published? Can I start to draft some implementation? Someone is taking this task and want some help?

@mttkay I saw that you commited on this "area" of the project... are you the person to ask about this stuff?

Thank you all for the attention and sorry about the large message :smile:
